sound/soc/sof/sof-priv.h

```diff
@@ -301,6 +301,7 @@ struct snd_sof_pcm_stream {
 	struct sof_ipc_stream_posn posn;
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
+	bool d0i3_compatible; /* DSP can be in D0I3 when this pcm is opened */
```

Comment:
fix the commit message as well.
Support running at D0i3 low-power mode or not -> can tolerate a transition to the D0i3 substate while opened (thus seen as 'active' by pm_runtime)